### ISSUE TITLE
.gd file look empty in 4.0 editor, but isn't

### ISSUE THREAD

**Issue by CorentinBrule** [godotengine/godot#69064]
### Godot version

v4.0.beta5.official [89a33d28f]

### System information

Windows 11

### Issue description

after convert a godot project from 3.5 to 4.0, an error occurs when launching the game :
`Constant value uses script from "res://avatar/Item.gd" which is loaded but not compiled.`, targeting the line where I import from a class : `const ItemClass = preload("res://avatar/Item.gd")` in a script. 
When I try to show this file ("/avatar/Item.gd") in editor, the file seems empty. But isn't in other text edit.
perhaps it is related to the content of the file, because I have errors when i paste the content of the script in editor from outside.

- 3.5 version of the code :
```
extends Node2D
class_name Item, "res://diamand_texture_21x21.png"
var avatar

export(String, "A",  "B", "X", "Y", "RB", "RT", "LB", "LT") var bouton_manette

func _ready():
	avatar = get_parent()
	assert(avatar is KinematicBody2D, "L'objet '" + name + "' n'est pas l'enfant de l'avatar !")
	if is_visible_in_tree() == false:
		set_process(false)
		set_physics_process(false)
```

- code after convertion to 4.0 :
```
extends Node2D
class_name Item, "res://diamand_texture_21x21.png"
var avatar

@export var bouton_manette # (String, "A",  "B", "X", "Y", "RB", "RT", "LB", "LT")


func _ready():
	avatar = get_parent()
	assert(avatar is CharacterBody2D) #,"L'objet '" + name + "' n'est pas l'enfant de l'avatar !")
	if is_visible_in_tree() == false:
		set_process(false)
		set_physics_process(false)
```

### Steps to reproduce

- open project "TEST-35' with godot 4.0, make the full conversion.
- or just open "TEST-40" with godot 4.0

try to start game, try to open Class.gd in script editor

### Minimal reproduction project

[test-35.zip](https://github.com/godotengine/godot/files/10078289/test-35.zip)
[test-40.zip](https://github.com/godotengine/godot/files/10078291/test-40.zip)

**Comment by CorentinBrule** [comment]
sorry I didn't see i was in 4.0beta5. The same process in 4.0beta6 does not create empty file bug. And once corrected with the 4.0 syntax, everything seems to work. I keep it open because I don't find other issue. If it helps

**Comment by akien-mga** [comment]
Duplicate of https://github.com/godotengine/godot/issues/68862
